lib/sidekiq_unique_jobs/orphans/ruby_reaper.rb

```diff
@@ -15,6 +15,9 @@ class RubyReaper < Reaper
       # @return [String] the suffix for :RUN locks
       RUN_SUFFIX = ":RUN"
       #
+      # @return [Integer] the maximum combined length of sidekiq queues for running the reaper
+      MAX_QUEUE_LENGTH = 1000
```

Comment:
If we wanted to make this a little smarter, we could make it a configuration for the reaper. Much like `reaper_timeout` and friends. Perhaps a `reaper_max_queue_size`?